packages/@cosmos-building-blocks/common/README.md

```diff
@@ -0,0 +1,18 @@
+# Welcome to your CDK TypeScript Construct Library project!
```

Comment:
delete this for now, will re-add int the doc PR

Reply:
done